modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -146,33 +146,25 @@ The following roles are available at the org level and these are assigned via in
 |
 |===
 
+== Instance SSO 
 
-== Log-in methods
+_Use as a login method for instances within Projects in this Org._
 
-Log-in methods are different for each SSO level.
-Administrators can configure a combination of one or more of the log-in methods.
+You can choose which projects are included during set up.
+Applies to authentication at the instance level meaning that the SSO login method is shown when a user tries to access an instance.
+Role mapping is a feature exclusive to Instance SSO.
```

Comment:
Is it really? Role mapping exists at org level too and you've listed a whole table about org-level roles and their privileges.
I think what you're trying to say is that role-mapping via SSO is only available for instance SSO?